README.md

```diff
@@ -50,32 +53,53 @@ repositories {
 
 ### Continuous Integration
 #### Travis CI
-XebiaLabs uses Travis CI for continuous integration. Go to [Travis CI](https://travis-ci.org) and log in with your GitHub account. After you log in, you'll see the community plugins being built. The build configuration is located in the `.travis.yml` file in each repository. More information is available at the [Travis CI documentation site](http://docs.travis-ci.com/).
+XebiaLabs uses Travis CI for continuous integration. Go to [Travis CI](https://travis-ci.org) and log in with your GitHub account. After you log in, you'll see the community plugins being built. The build configuration is provided by the `.travis.yml` file in each repository. More information is available at the [Travis CI documentation site](http://docs.travis-ci.com/).
 
 Please also configure [HipChat](https://hipchat.com/) notifications as described [here](http://docs.travis-ci.com/user/notifications/#HipChat-notification) (TravisCI room) and GitHub releases as described [here](http://docs.travis-ci.com/user/deployment/releases/).
 
 #### Circle CI
 As an alternative, you can use Circle CI.
 
-### Use a Gradle wrapper
-Using a Gradle wrapper ensures that the same Gradle version is used (including on Travis CI). An example usage of a Gradle wrapper can be found in [the xld-openshift-plugin repository](https://github.com/xebialabs-community/xld-openshift-plugin). You can copy the example wrapper or create one yourself using `gradle wrapper`.
+### Gradle Wrapper
+Using a Gradle wrapper ensures that the same Gradle version is used for all builds (including on Travis CI). An example Gradle wrapper can be found in [the xld-openshift-plugin repository](https://github.com/xebialabs-community/xld-openshift-plugin). You can copy the example wrapper or create one yourself using `gradle wrapper`.
+
+Builds should typically be ran with `./gradlew build`.
 
-### Release your plugin (general outline)
-Make sure you follow these steps
+## Release Outline
+
+### Versioning - Manual/Basic
 
 * Update the `build.gradle` file to contain the new version number.
 * Update the `.travis.yml` file to contain the new version number.
-* Create a new tag. For example: `git tag -a v4.5.2 -m 'Version 4.5.2' 373159`.
+* Create a new tag, using [semantic versioning](https://semver.org). For example: `git tag -a v4.5.2 -m 'Version 4.5.2' 373159`.
```

Comment:
I don't know if that trailing 373159 is actually needed.  In one of my commits, I did not include a trailing number and the command worked just fine.

Reply:
Agreed - don't believe it's needed.

Reply:
Actually it could be - if someone wants to tag a different commit than the most recent one, for use in the release.  I've never done that personally, but seems like some might.  I'll leav it.

Reply:
Went with a hybrid approach.  Thanks for the note.